.github/renovate.json

```diff
@@ -0,0 +1,6 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "github>containers/automation//renovate/defaults.json5"
+  ]
+}
```

Comment:
Would you mind making this a `.json5` file, mainly because that allows adding comments :grin: 

While you're at it, would you mind including something like the following:

```json5
/*
   Configuration Reference:
   https://docs.renovatebot.com/configuration-options/

   Monitoring Dashboard:
   https://app.renovatebot.com/dashboard#github/containers

   Configuration Update/Change Procedure:
     1. Make changes
     2. Manually validate changes (from repo-root):

        podman run -it \
            -v ./.github/renovate.json5:/usr/src/app/renovate.json5:z \
            ghcr.io/renovatebot/renovate:latest \
            renovate-config-validator
     3. Commit.
*/
```

Reply:
Changed the file type to json5, added the requested comments and squashed the commit in [54d9214](https://github.com/containers/crun-vm/pull/64/commits/54d921467df053dd012dfaf17286a6046870f6a4).

If preferred, I can add a super simple workflow that validates the Renovate configuration which runs this automatically?  

```yaml
name: Renovate

on:
  pull_request:
    paths:
      - ".github/renovate.json"
      - ".github/renovate.json5"
      - "renovate.json"
      - "renovate.json5"

jobs:
  validate:
    runs-on: ubuntu-latest
    steps:
      - name: Checkout
        uses: actions/checkout@a5ac7e51b41094c92402da3b24376905380afc29 # v4

      - name: Setup Node.js
        uses: actions/setup-node@60edb5dd545a775178f52524783378180af0d1f8 # v4
        with:
          node-version: latest

      - name: Install dependencies
        run: npm install -g renovate

      - name: Validate Renovate config
        run: renovate-config-validator --strict
```

It doesn't do caching or anything fancy, but I find it faster than the majority of other tests so that it won't be a bottleneck.

It can even be a reusable workflow or action so each repo doesn't need to contain any Renovate validation logic.

Reply:
> I can add a super simple workflow

It's not a half-bad idea actually.  If you have time, feel free to do that in a separate PR.  No reason to hold this one up for it.

Reply:
https://github.com/marketplace/actions/validate-renovate-configuration-with-renovate-config-validator ?

Reply:
I was just going to DIY it :smile: 

They provide the Docker image you shared, so can either use that in a workflow, or as above by installing the NPM package.

It seems too simple to need an external dependency IMO.